test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -349,6 +462,48 @@ public void ConfigGetCommand_NullGetLogger_Fail()
             Assert.Throws<ArgumentNullException>(() => ConfigGetRunner.Run(args, null));
         }
 
+        [Fact]
+        public void ConfigSetCommand_InvalidConfigKey_Fail()
+        {
+            // Arrange & Act
+            using var testInfo = new TestInfo("NuGet.Config");
+            var key = "InvalidConfigKey123";
+            var value = "https://TestRepo2/ES/api/v2/package";
+            var filePath = Path.Combine(testInfo.WorkingPath, "NuGet.Config");
+
+            var result = CommandRunner.Run(
+                DotnetCli,
+                Directory.GetCurrentDirectory(),
+                $"{XplatDll} config set {key} {value} --configfile {filePath}",
+                waitForExit: true
+                );
+            var expectedError = string.Format(CultureInfo.CurrentCulture, Strings.Error_ConfigSetInvalidKey, key);
+            // Assert
+            DotnetCliUtil.VerifyResultFailure(result, expectedError);
+        }
+
+        [Fact]
+        public void ConfigSetCommand_NullArgs_Fail()
```

Comment:
Can you rename the test methods to follow the pattern described in the coding guidelines:
https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md#unit-test-method-naming

Specifically:
`Format: <what is being tested>_<under what conditions>_<with what expectations>`

With that pattern, the suffixes of _Fail and _Success should be more descriptive of what the expectation is of how they succeed or fail.